lib/features/composer/presentation/composer_controller.dart

```diff
@@ -688,7 +689,7 @@ class ComposerController extends BaseController with DragDropFileMixin implement
       if (composerArguments.value?.emailActionType == EmailActionType.reopenComposerBrowser) {
         log('ComposerController::_initEmail: hasRequestReadReceipt = ${arguments.hasRequestReadReceipt}');
         hasRequestReadReceipt.value = arguments.hasRequestReadReceipt ?? false;
-      } else {
+      } else if (composerArguments.value?.emailActionType != EmailActionType.editDraft) {
```

Comment:
why we need to getSettings in this case? IMO, we only need to read from the draft and add it to composer (dont care about setting here)